src/main/java/org/folio/dew/service/ErrorMessage.java

```diff
@@ -0,0 +1,13 @@
+package org.folio.dew.service;
+
+import lombok.Getter;
+
+@Getter
+public class ErrorMessage {
+
+  private String value;
+
+  public void setValue(String value) {
+    if (this.value == null) this.value = value;
```

Comment:
Why do you need this validation? In all places where this method is used, the argument `value` cannot be `null`.

Reply:
To exclude this validation and make code clearer it is enough to change processing of `ErrorMessage` processing in the method `private ItemFormat applyContentUpdate(ItemFormat itemFormat, ContentUpdate contentUpdate, ErrorMessage errorMessage)`. For example, we can set variable `errorMessage` to `null` instead of using `ErrorMessage` with null-value.

Reply:
Hi @khandramai , we need to this just to be sure that only one error - more specific will appear per record.  That error will be logged to the error file

Reply:
@alekGbuz  , from the bean implementation point of view null-validation for setter looks strange. Setter should change the state of the bean but it shouldn't contain any logic that affect business flow.